### ISSUE TITLE
Double-braces incorrectly trigger braces/min-spaces-inside

### ISSUE THREAD

**Issue by sbrowne-godaddy** [adrienverge/yamllint#781]
When using the following snippet of `.yamllint`:
```yaml
---
extends: default
  braces:
    min-spaces-inside: 1
    max-spaces-inside: 1
```
A YAML file that has a value with double braces incorrectly triggers:
```yaml
---
keyname: {{ value }}
```
The output is as such:
```sh
[user@laptop sceptre (master)]$ ~/.cache/pre-commit//repo7_jyo6cg/py_env-python3.13/bin/yamllint test.yaml 
test.yaml
  2:11      error    too few spaces inside braces  (braces)
  2:20      error    too few spaces inside braces  (braces)
```

A quick flick through the spec doesn't immediately reject the use of `{{`.

In particular, Sceptre (for AWS CloudFormations) make use of this form of double-brace.

It would be nice if this could either be handled by default or a new rule could be created to allow for it to treat them as single-brace (for linting purposes).

```sh
[user@laptop sceptre (master)]$ ~/.cache/pre-commit//repo7_jyo6cg/py_env-python3.13/bin/yamllint --version
yamllint 1.37.1
```

**Comment by adrienverge** [comment]
Hello Stuart,

Your example:
```yaml
---
keyname: {{ value }}
```
is not valid YAML: `{{` is forbidden because a `{` starts a mapping and should be followed by a key-value pair, or an end of mapping (`}`). Parsing with a parser (for instance PyYAML) fails:
```
Traceback (most recent call last):
  …
    raise ConstructorError("while constructing a mapping", node.start_mark,
            "found unhashable key", key_node.start_mark)
yaml.constructor.ConstructorError: while constructing a mapping
```

I guess this snippet is a Jinja2 template, meant to be transformed into valid YAML by Sceptre. It pre-processes templates to transform `{{ … }}` and `{% if … %}` and produce a valid YAML document.

See similar cases in already-existing issues: #61, #65, #128, #311, #408, #460, #462, #507, #531, #547…
In those issues you'll find workarounds that work in *some* cases. In your precise case, quoting the value could work:
```yaml
---
keyname: '{{ value }}'
```
